exercise/staff_views.py

```diff
@@ -84,7 +88,7 @@ class IncreaseSubmissionMaxView(SubmissionMixin, BaseRedirectView):
     access_mode = ACCESS.GRADING
 
     def post(self, request, *args, **kwargs):
-        deviation,_ = MaxSubmissionsRuleDeviation.objects.get_or_create(
+        deviation, _ = MaxSubmissionsRuleDeviation.objects.get_or_create(
```

Comment:
The PR should not include irrelevant coding style changes in places that are not even otherwise modified.